### PR TITLE
[15.0][MIG] stock_barcodes_gs1

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

Depends on:
 - https://github.com/OCA/stock-logistics-barcode/pull/404

### CODE REVIEW

File: stock_barcodes_gs1/wizard/stock_production_lot.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    def _decode_barcode(self, barcode):
+        return self.env["gs1_barcode"].decode(barcode)
+
+    def on_barcode_scanned(self, barcode):
+        try:
+            barcode_decoded = self._decode_barcode(barcode)
+        except Exception:
+            return super().on_barcode_scanned(barcode)
+        package_barcode = barcode_decoded.get("01", False)
```
Comment (rousseldenis) [inline stock_barcodes_gs1/wizard/stock_production_lot.py:18]: Shouldn't the type of barcode depends on nomenclature ?

@sergio-teruel @JasminSForgeFlow

Comment (simahawk) [reply stock_barcodes_gs1/wizard/stock_production_lot.py:18]: Agree, see https://github.com/OCA/stock-logistics-barcode/pull/445

**Comment by sergio-teruel** [comment]
Can you get missed commits from 14?

**Comment by pedrobaeza** [comment]
Superseded by #497

**Pull request closed without merging**